provider/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptTarget.kt

```diff
@@ -77,13 +82,16 @@ fun projectScriptTarget(project: Project) =
 internal
 data class KotlinScriptTarget<T : Any>(
     val `object`: T,
+    val rootDir: File,
```

Comment:
:x: `rootDir` feels out of place here. I'd move it after `pluginsBlockTemplate` and before `accessorsClassPath`.

Reply:
Good call, fixed